### PR TITLE
Removed unwanted panel graph header

### PR DESCRIPTION
**Pull request by InsaneSplash**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

File: html/pages/graphs.inc.php [hunk 1]
```diff
@@ -71,7 +71,6 @@
 
     print_optionbar_end();
 
-    print_optionbar_start();
```
Comment (laf) [inline html/pages/graphs.inc.php]: This leaves a rogue `print_optionbar_end()`. Are you sure it needs removing?

Comment (InsaneSplash) [reply html/pages/graphs.inc.php]: Yes, the one above this line is for the normal panel header... it seems there was a start() added, but never had an end.... It also changed the mini-graphs in the view to have a black background with the mono theme....

Comment (InsaneSplash) [reply html/pages/graphs.inc.php]: let me revisit this.... as I found one way down in the code..... very weird.... and I dont think its needed to be honest....

Comment (laf) [reply html/pages/graphs.inc.php]: What I mean is if you search for print_optionbar_ in the file, you now have 5 entries where as surely it should be 4 or 6? 3 starts and 3 ends or 2 of each?

Comment (InsaneSplash) [reply html/pages/graphs.inc.php]: Should be good now...

Comment (InsaneSplash) [reply html/pages/graphs.inc.php]: BEFORE:
image

AFTER:
image

**Pull request merged**